_posts/2018-03-08-roadmap.md

```diff
@@ -0,0 +1,221 @@
+  produce static binaries come together to great effect for writing CLI apps.
+- **WebAssembly**. The “wasm” web standard allows shipping native-like binaries
+  to all major browsers, but GC support is still years away. Rust
+  is [extremely well positioned](https://mgattozzi.com/rust-wasm) to target this
+  domain, and provides a reasonable on-ramp for programmers coming from JS.
+- **Embedded devices**. Rust has the potential to make programming
+  resource-constrained devices much more productive—and fun! We want embedded
+  programming to reach first-class status this year.
+
+Each of these domains has a dedicated working group for the year. These WGs will
+work in a cross-cutting fashion, interfacing with language, tooling, library,
+and documentation work.
+
+## Better serving intermediate Rustaceans
+
+One of the strongest messages we’ve heard from production users, and [the 2017
+survey], is that people need more resources to take them from understanding
+Rust’s concepts to knowing how to use them _effectively_. The roadmap does not
+stipulate exactly what these resources should look like
+— [probably there should be several kinds][intermediate] — but commits us as a
+community to putting significant work into this space, and ending the year with
+some solid new material.
+
+[the 2017 survey]: https://blog.rust-lang.org/2017/09/05/Rust-2017-Survey-Results.html
+[intermediate]: https://quietmisdreavus.net/code/2018/01/10/not-a-layer-cake-analogy/
+
+## Community
+
+**Connect and empower Rust's global community**. We will pursue
+internationalization (specifically: translation) as a first-class concern, and
+proactively work to build ties between Rust subcommunities currently separated
+by language, geography, or culture. We will spin up and support Rust events
+worldwide, including further growth of the RustBridge program.
+
+**Grow Rust's teams and new leaders within them**. We will refactor the Rust
+team structure to support more scale, agility, and leadership growth. We will
+systematically invest in mentoring, both by creating more on-ramp resources and
+through direct mentorship relationships.
+
+## A call to action
+
+As always in the Rust world, the goals laid out here will ultimately be the
+result of a community-wide effort—maybe one including you! Here are some of the
+teams where we could use the most help. Note that all IRC channels refer to the
+irc.mozilla.org network.
+
+- **WebAssembly WG**. Compiling Rust to WebAssembly should be _the_ best choice for fast code on the Web. Check out [rust-lang-nursery/rust-wasm](https://github.com/rust-lang-nursery/rust-wasm) to learn more and get involved!
+- **CLI WG**. Writing CLI apps in Rust should be a frictionless experience  – from finding the right libraries and writing concise integration tests up to cross-platform distribution. Join us at [https://github.com/rust-lang-nursery/cli-wg](https://github.com/rust-lang-nursery/cli-wg) and help us reach that goal!
```

Comment:
Can you fix the double space and adjust the link to `[rust-lang-nursery/cli-wg](https://github.com/rust-lang-nursery/cli-wg)`? Thanks! :)